src/send/Send.tsx

```diff
@@ -221,9 +221,7 @@ function Send({ route }: Props) {
       <SendHeader isOutgoingPaymentRequest={isOutgoingPaymentRequest} />
       <DisconnectBanner />
       <SendSearchInput input={searchQuery} onChangeText={throttledSearch} />
-      {inviteRewardsEnabled && numberVerified && hasGivenContactPermission && (
-        <InviteRewardsBanner />
-      )}
+      {inviteRewardsEnabled && hasGivenContactPermission && <InviteRewardsBanner />}
```

Comment:
the `inviteRewardsEnabled` selector already takes into account phone verification, we shouldn't show this banner to non CPV users anyway